### DISCUSSION TITLE
Connect to a database in JointJs plus

### DISCUSSION THREAD

**Discussion by AngelDavid** [clientIO/joint#2128] [Q&A]
### Introduction

Hi all,  as Iam new to jointjs, I was analyzing the features of JointJs and JointJs plus. I have few queries in this. 
* Can we connect with database to create chart in JointJs?  Is it also available in JointJs plus? 
* if yes,  what are the different databases supported? 
* Can we retrieve data from excel spreadsheet and use them as database?
*
Kindly let me know your thoughts on this.. 

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
*JointJS* is a client-side JavaScript library. It supports export and import of diagrams in [JSON](https://resources.jointjs.com/tutorial/serialization) format  (*JJ+* also [vsdx](https://resources.jointjs.com/docs/rappid/v3.6/format.Visio.html)).

It is entirely up to you how you implement data persistence, e.g. which database you choose - it should be able to store a file, JSON or a string.
